sdk/src/azure/core/az_mqtt5_rpc_client.c

```diff
@@ -10,100 +10,65 @@
 
 AZ_NODISCARD az_mqtt5_rpc_client_options az_mqtt5_rpc_client_options_default()
 {
-  return (az_mqtt5_rpc_client_options){ .subscribe_timeout_in_seconds
-                                        = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
-                                        .publish_timeout_in_seconds
-                                        = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS };
+  return (az_mqtt5_rpc_client_options){
+    .subscribe_timeout_in_seconds = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
+    .publish_timeout_in_seconds = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
+    .subscription_topic_format
+    = AZ_SPAN_FROM_STR("vehicles/{serviceId}/commands/{executorId}/{name}/__for_{invokerId}\0"),
```

Comment:
It feels like this mixes a sample with the actual SDK. Maybe the default formats should be AZ_SPAN_EMPTY?

Reply:
that works for me! If there ends up being a default topic format in the future, we could always add that back in.

Reply:
changed to AZ_SPAN_EMPTY and added setting the formats in all samples